### ISSUE TITLE
fidesctl fails to start webserver with 1.3.1.

### ISSUE THREAD

**Issue by iokamoto-sm** [ethyca/fides#378]
### Bug Description

Installed fidesctl 1.3.1 with pip. (i.e. `pip install fidesctl[webserver]==1.3.1`) `fidesctl webserver` fails to start the webserver with the following error.

```bash
root@78d1b8b390e0:/opt/fides/fidesctl2# fidesctl webserver
Packages not found, try: pip install "fidesctl[webserver]"
```

#### Steps to Reproduce

1. install fidesctl with pip
`pip install fidesctl[webserver]==1.3.1`
2. create config file
`fidesctl init`
3. start webserver
`fidesctl webserver`

It raises the error reported in the description.

### Expected behavior

It should find `fidesctl[webserver]` and start it successfully.

### Screenshots

n.a.

### Environment

* Version: 1.3.1
* OS: MacOS 12.2.1
* Python Version: 3.9.10
* Docker Version: 20.10.8, build 3967b7d

### Additional context

n.a.

**Comment by ThomasLaPiana** [comment]
@iokamoto-sm thank you for the bug report!

I was able to reproduce this with 1.3.1, but with the current version on `main` this should be fixed. Can you try installing the version on `main` and letting me know if you still run into this issue?

**Comment by iokamoto-sm** [comment]
Hi @ThomasLaPiana , I am actually seeking a fix with pip package. I am preparing the docker image with fidesctl so other developers can easily use fidesctl. While I do so, I need pip package works properly. It would be great if fides can update pip package with the fix.

**Comment by ThomasLaPiana** [comment]
@iokamoto-sm understood, the pip package will be fixed once 1.4.0 is released in the near future. I was just looking for confirmation from you that the current version in `main` works for you, which will confirm that the pip release for 1.4.0 release will work as well.
